.kokoro/tests/run_tests.sh

```diff
@@ -92,8 +100,27 @@ for file in **/requirements.txt; do
     file=$(dirname "$file")
     cd "$file"
 
+    # First we look up the environment variable `RUN_TESTS_DIRS`. If
+    # the value is set, we'll iterate through the colon separated
+    # directory list.
+    if [[ -n "${RUN_TESTS_DIRS:-}" ]]; then
+	IFS=":"
+	read -ra run_tests_dirs <<< "${RUN_TESTS_DIRS}"
+	match=0
+	for d in "${run_tests_dirs[@]}"; do
+	    # If the current dir starts with one of the
+	    # RUN_TESTS_DIRS, we should run the tests.
+	    if [[ "${file}" = "${d}"* ]]; then
+		match=1
+		break
+	    fi
+	done
+	IFS=" "
+	if [[ $match -eq 0 ]]; then
+	    continue
+	fi
```

Comment:
suggestion: Would this be simpler with a regex?

```bash
if [[ ! "$RUN_TESTS_DIRS" =~ ^(.*:)?("$file"[^:]*)(:.*)?$ ]]; do
    continue
elif
```


If you prefer not to use regexs, how about `tr`? 

```bash
for d in $(echo $RUN_TESTS_DIRS | tr ";" "\n"); do
   // loop stuff
done
```

Reply:
I chose `tr`